### ISSUE TITLE
[주제] 질문

### ISSUE THREAD

**Issue by pseeej** [sangmandu/SangSangPlus#85]
[주제] 에 대한 질문입니다.

진행일 : 2021.xx.xx

**Comment by sangmandu** [comment]
안녕하세요

**Comment by sangmandu** [comment]
> 안녕하세요

이렇게돼요

**Comment by sangmandu** [comment]
1. 질문제목
2. 질문제목
3.

**Comment by sangmandu** [comment]
## [질문] 3장 Huggingface 실습 중 attention mask🔴
![image](https://user-images.githubusercontent.com/45033215/135115924-7f91616f-7f9a-498d-b389-9890d0b6ba62.png)
attention_mask 설명이 좀 빈약해서 검색해봤는데도 잘 모르겠네요. 어떤 역할이고 결과값은 왜 저렇게 출력되었는지 아시나요?

**Comment by raki-1203** [comment]
[질문] 논문 1. Introduction 에서 마지막 contributions 중에서 해석 질문 🔴

![image](https://user-images.githubusercontent.com/52475378/135212475-6039bdaa-59ba-4fa3-b5bb-5f385378c254.png)

여기서 중간에 나오는 suite 를 어떻게 받아들이면 될까요?
task? dataset?

**Comment by raki-1203** [comment]
[질문] 논문 3. BERT 부분 4장에서 Input/Output Representations 이 문장의 의미 질문 🔴

![image](https://user-images.githubusercontent.com/52475378/135212777-4a63465d-294a-4888-b8d8-e368e6c2bf27.png)

이 부분을 어떻게 해석하셨을 까요?

**Comment by raki-1203** [comment]
## 논문 3.1 Pre-training BERT Task #2: Next Sentence Prediction (NSP) 부분 질문 🔴

![image](https://user-images.githubusercontent.com/52475378/135213478-72180e0d-05ea-4d4c-b3fb-021b1e0ebb05.png)

Jernite et al. (2017) 과 Logeswaran and Lee (2018) 여기 두 논문에서 사용된 objectives 가 무엇인가요?

**Comment by whatchang** [comment]
## [질문] 논문 3. BERT 부분 4장에서 Input/Output Representations 이 문장의 의미 질문 🟢 

![스크린샷 2021-09-29 오후 6 27 32](https://user-images.githubusercontent.com/22788924/135242290-3c262168-70c6-4195-a06c-3b6ec81246a5.png)

fine-tuning에서도 [MASK] 심볼을 사용하나요?

## [답변] "symbol never appears during the fine-tuning stage"를 통해서 fine-tuning에서 해당 심볼을 사용하지 않음을 알 수 있습니다.

**Comment by SinGonKim** [comment]
[질문]BERT Large의 경우 소규모 데이터 셋에서 fine-tuning이 불안정한 이유🟢
질문해보겠습니다.
![1](https://user-images.githubusercontent.com/87477828/135268804-e8cbd22f-866d-47e1-be85-5d59a89a82cf.png)

[답변] 멘토님
적은 데이터이기 때문에 over-fitting이 되서 차라리 성능이 약간 안좋더라도 over-fitting이 안되는 모델을 쓰는 게 낫지 않을까 이야기지 무조건 BERT Large가 다른  BERT Base나 다른 모델들에 비해서 좋다.
